arcaflow_plugin_sysbench/sysbench_schema.py

```diff
@@ -433,6 +626,37 @@ class SysbenchCpuResultParams:
     ]
 
 
+@dataclass
+class SysbenchIoResultParams:
+    """
+    This is the output results data structure for sysbench io results.
+    """
+
+    Fileoperations: typing.Annotated[
+        FileOperationMetrics,
+        schema.name("FileOperations"),
+        schema.description("File Operation Metrics"),
+    ]
+    Throughput: typing.Annotated[
+        ThroughputMetrics,
+        schema.name("Throughput"),
+        schema.description("Throughput metrics"),
+    ]
+    Latency: typing.Annotated[
+        LatencyAggregates,
+        schema.name("Latency"),
+        schema.description("CPU latency in miliseconds"),
```

Comment:
Is this actually CPU latency? Or file operation latency?

Reply:
I've not clue about that, what I know is that the output structure is the same as for the other tests, so I supposed it was the same kind of "Latency".